react-frontend/src/components/Home/cardlist.js

```diff
@@ -16,10 +16,6 @@ const descriptions = [
 const imgLinks = [
   'https://dalehartery.files.wordpress.com/2015/05/light-grey-background-pattern.jpg',
```

Comment:
i'm hoping this will be temporary. Downloading images off the internet from a bros wordpress is not ideal we have no control when its deleted.